app/indices/answers_index.rb

```diff
@@ -0,0 +1,7 @@
+ThinkingSphinx::Index.define :answer, with: :active_record do
+  indexes body, sortable: true
+  indexes user.profile.first_name, as: :author_first_name, sortable: true
```

Comment:
You can use delegation

Reply:
> You cannot reference model methods with SQL-backed indices - in this case, Sphinx talks directly to your database, and Ruby doesn’t get loaded.

I think, this is right, because `active_record` index used for this.